llvm/test/CodeGen/X86/fold_bitcast_md_range.ll

```diff
@@ -0,0 +1,24 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=x86_64-apple-macosx10.12.0 -O0 < %s | FileCheck %s
```

Comment:
```suggestion
; RUN: llc -mtriple=x86_64-apple-macosx10.12.0  < %s | FileCheck %s
```

Reply:
-O0 is unnecessary.  Verified crash without -O0 and without fix.  With fix test passes.